src/services/page-service.ts

```diff
@@ -0,0 +1,27 @@
+import config from 'config';
+import { FindManyOptions } from 'typeorm';
+
+import { Search } from '../entities/common-interfaces';
+
+const { pageNumberDefault, pageSizeDefault } = config.get('paging');
+
+interface FindOptions<T> extends FindManyOptions<T> {
+  number: number;
+  size: number;
+}
+
+export const parsePageParams = <T>(query: Search): FindOptions<T> => {
+  const { pageNumber, pageSize, sortingColumn, sortingDirection = 'ASC' } = query;
```

Comment:
just for consistency you could move `ASC` to `config` and desctructure with renaming and assigning defaults to `number` and `size` in the current statement